lib/core/versions/latest/AnchorFile.ts

```diff
@@ -103,4 +111,15 @@ export default class AnchorFile {
 
     return false;
   }
+
+  /**
+   * Creates a buffer from the input so that the buffer can be persisted.
+   */
+  public static async createBufferFromAnchorFileModel (anchorFileModel: AnchorFileModel): Promise<Buffer> {
```

Comment:
Consider rename method to `createCompressedBufferFromAnchorFileModel` or maybe just `compressAnchorFileModel`

Reply:
I feel like that 'compressing' information is an internal detail and should not be exposed to caller. Tomorrow, the AnchorFile.ts can decide to not compress and that detail should stay within.